genie-web/src/test/java/com/netflix/genie/web/rpc/grpc/services/impl/v4/GRpcJobTerminationServiceImplUnitTests.java

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
All other new tests are in Spock, any particular reason for going with JUnit?

Reply:
This comment is on `GRpcJobTerminationServiceImplUnitTests.java` which you added.
And I was wondering why it is JUnit and not Spock